app/models/service_template/copy.rb

```diff
@@ -7,6 +7,9 @@ def template_copy(new_name = "Copy of " + name + Time.zone.now.to_s)
         dup.tap do |template|
           template.update_attributes(:name => new_name, :display => false)
           service_resources.each { |service_resource| resource_copy(service_resource, template) }
+          resource_actions.each { |resource_action| resource_action_copy(resource_action, template) }
+          picture_copy(template) if picture.present?
```

Comment:
Is the `.present?` necessary?  `picture` should be a Picture instance or `nil`, right?